pom.xml

```diff
@@ -74,9 +76,7 @@
     <dokka.version>1.5.0</dokka.version>
 
     <!-- Import-Package definitions for maven-bundle-plugin -->
-    <osgi.import.package.root>
-      *
-    </osgi.import.package.root>
+    <osgi.import.package.root>*</osgi.import.package.root>
```

Comment:
Isn't `com.querydsl.*` a more sensible default then `*` here?

Reply:
no, '*' is default for bnd and it means that bnd imports all referred packages